### PR TITLE
Implicit topics that can be used to indicate action topics (fixes #101)

### LINKED ISSUE #101: Topics generated by actions should be indicated somehow
Currently, when actions are created they also generate the topics that are implied by the actions. When processing the architecture, this causes both actions and topics to be created in a node. Only the action is needed.

### PR DESCRIPTION
**Pull request by schmerl**
See #101

### CODE REVIEW

File: src/rosdiscover/interpreter/topic.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+__all__ = ('Topic',)
```
Comment (ChrisTimperley) [inline src/rosdiscover/interpreter/topic.py:2]: Missing whitespace below this line

File: src/rosdiscover/interpreter/topic.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+from typing import Any, Dict
```
Comment (ChrisTimperley) [inline src/rosdiscover/interpreter/topic.py:3]: Missing whitespace

File: src/rosdiscover/interpreter/topic.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+import attr
+
+
+@attr.s(frozen=True, slots=True)
```
Comment (ChrisTimperley) [inline src/rosdiscover/interpreter/topic.py:7]: This is great. If you want to make it even tidier, you can add `auto_attribs=True` to the `attr.s` call and drop ` = attr.ib()` from each attribute. (See: https://github.com/ChrisTimperley/rosdiscover/blob/master/src/rosdiscover/interpreter/summary.py#L9-L12)

**Comment by schmerl** [comment]
I think this can be merged now.

**Pull request merged**